renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "prConcurrentLimit": 3,
+  "packageRules": [
+    {
+      "description": "Automerge non-major updates in .tekton folder to development branch",
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "matchFileNames": [".tekton/**"],
```

Comment:
Maybe we could also include `.pre-commit-config.yaml`, since it can generate multiple PRs as well. It's executed as part of the automatic linting process, so it should be fairly safe to include. What do you think?